impexp-client-gui/src/main/resources/org/citydb/gui/icons/code.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="16px" viewBox="0 0 24 24" width="16px" fill="#6E6E6E"><path d="M0 0h24v24H0V0z" fill="none"/><path d="M9.4 16.6L4.8 12l4.6-4.6L8 6l-6 6 6 6 1.4-1.4zm5.2 0l4.6-4.6-4.6-4.6L16 6l6 6-6 6-1.4-1.4z"/></svg>
```

Comment:
Is this a Google Material Icon (https://fonts.google.com/icons?selected=Material+Icons)?

Reply:
yes